packages/space-opera/editor/index.html

```diff
@@ -86,50 +103,57 @@
   }
 
   me-tabs {
-    background-color: #202124;  /* GOOGLE_GREY_900 */
+    background-color: var(--expandable-section-background);
     padding: 0;
-    width: 360px;
+    width: 440px;
+    border-left: 1px solid var(--tabs-conatiner-border)
   }
 
   me-tabs a {
     color: #dadcff; /* GOOGLE_GREY_300, but.. bluer */
   }
 
   me-tabbed-panel {
-    max-height: calc(100vh - 96px);
+    max-height: calc(100vh - 45px);
     overflow-y: auto;
     display: block;
   }
 </style>
 <body>
-  <model-editor>
-    <div class="app">
+<model-editor>
+  <div class="app">
     <div class="editor-body-root">
       <div class="mvContainer">
         <model-viewer-preview id="editing_adapter">
         </model-viewer-preview>
       </div>
       <me-tabs>
-        <me-tabbed-panel icon="save">
-          <model-viewer-snippet></model-viewer-snippet>
+        <me-tabbed-panel icon="home">
+          <home-container></home-container>
+        </me-tabbed-panel>
+        <me-tabbed-panel icon="import_export">
+          <me-export-panel></me-export-panel>
         </me-tabbed-panel>
         <me-tabbed-panel icon="create">
+          <me-export-panel header="true"></me-export-panel>
           <me-transformation-controls></me-transformation-controls>
           <me-ibl-selector></me-ibl-selector>
           <me-animation-controls></me-animation-controls>
           <me-hotspot-panel></me-hotspot-panel>
           <me-poster-controls></me-poster-controls>
         </me-tabbed-panel>
-        <me-tabbed-panel icon="color_lens">
-          <me-materials-panel></me-materials-panel>
-        </me-tabbed-panel>
         <me-tabbed-panel icon="photo_camera">
+          <me-export-panel header="true"></me-export-panel>
           <me-camera-settings></me-camera-settings>
         </me-tabbed-panel>
+        <me-tabbed-panel icon="color_lens">
+          <me-materials-panel></me-materials-panel>
```

Comment:
Materials tab UX feedback:
![image](https://user-images.githubusercontent.com/1649964/99306212-b7e53280-2809-11eb-9bd4-ff6243c404e1.png)

Reply:
Mostly, let's get consistent ordering and minimize the vertical space used by stacking things more horizontally.

Reply:
The texture and color picker being horizontal with the header and revert button just doesn't look good. I know it adds additional vertical space, but the alternative is a really janky looking row of things of multiple sizes. If I make everything the same size, then it gets even more awkward with text too big or buttons too small. I think this is something we can live with for the time being.

Reply:
Can you just revert it to how it was originally? I this case I feel like the original editor had a cleaner layout.